extensions/px-to-rem-converter/package.json

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "px-to-rem-converter",
+  "title": "PX to Rem converter",
+  "description": "Conversions between PX and REM",
+  "icon": "icon.png",
+  "author": "raulanatol",
+  "categories": [
+    "Design Tools",
+    "Developer Tools",
+    "Productivity",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "px2rem",
+      "subtitle": "PX to REM converter",
+      "description": "Conversions between PX and REM",
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "basePixel",
+          "type": "textfield",
+          "required": false,
+          "title": "base px value",
```

Please use title case for user-facing strings:

```suggestion
          "title": "Base PX Value",
```